app/controllers/parks_controller.rb

```diff
@@ -12,6 +12,10 @@ def edit
     @park = Park.find(params[:id])
   end
 
+  def create
+    redirect_to parks_path(params.park_id)
+  end
```

Comment:
yipe. I have no idea why your create method is doing that, but it doesnt look fun. Are you still using this at all? I thought your API::ParksController handles creation of Parks. If that is so, you should remove this altogether